simulation/halsim_ws_core/src/main/native/cpp/WSProvider_PWM.cpp

```diff
@@ -28,11 +28,8 @@ HALSimWSProviderPWM::~HALSimWSProviderPWM() {
 
 void HALSimWSProviderPWM::RegisterCallbacks() {
   m_initCbKey = REGISTER(Initialized, "<init", bool, boolean);
-  m_speedCbKey = REGISTER(Speed, "<speed", double, double);
-  m_positionCbKey = REGISTER(Position, "<position", double, double);
   m_rawCbKey = REGISTER(PulseMicrosecond, "<raw", int32_t, int);
-  m_periodScaleCbKey = REGISTER(PeriodScale, "<period_scale", int32_t, int);
-  m_zeroLatchCbKey = REGISTER(ZeroLatch, "<zero_latch", bool, boolean);
+  m_outputPeriodCbKey = REGISTER(OutputPeriod, "<output_period", int32_t, int);
```

Comment:
Need to update the websocket doc to match (both hardware_ws_api.md and wpilib-ws.yaml).  Note this will break the Romi motor interface as well, but that should probably be changed to use a SimDevice.

Reply:
Theres already a decent chunk of romi breakages in 2027. But I'll update that for now.

Reply:
Fixed